tracing/src/macros.rs

```diff
@@ -1959,6 +1959,52 @@ macro_rules! valueset {
             $next,
         )
     };
+
+    // TODO(#1138): determine a new syntax for uninitialized span fields, and
+    // re-enable this.
+    // (@{ $(,)* $($out:expr),* }, $next:expr, $k:literal = _, $($rest:tt)*) => {
+    //     $crate::valueset!(@ { $($out),*, (&$next, None) }, $next, $($rest)*)
+    // };
```

Comment:
This is no longer necessary --- disabled fields can be created with `field::Empty`. We should probably not copy-paste this comment any more (and I should, eventually, remove the original).

Reply:
Removed